<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>测试报告</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    
<style type="text/css" media="screen">
body        { font-family: verdana, arial, helvetica, sans-serif; font-size: 80%; }
table       { font-size: 100%; }
pre         { }

/* -- heading ---------------------------------------------------------------------- */
h1 {
	font-size: 16pt;
	color: gray;
}
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}

.heading .attribute {
    margin-top: 1ex;
    margin-bottom: 0;
}

.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}

/* -- css div popup ------------------------------------------------------------------------ */
a.popup_link {
}

a.popup_link:hover {
    color: red;
}

.popup_window {
    display: none;
    position: relative;
    left: 0px;
    top: 0px;
    /*border: solid #627173 1px; */
    padding: 10px;
    background-color: #E6E6D6;
    font-family: "Lucida Console", "Courier New", Courier, monospace;
    text-align: left;
    font-size: 8pt;
    width: 500px;
}

}
/* -- report ------------------------------------------------------------------------ */
#show_detail_line {
    margin-top: 3ex;
    margin-bottom: 1ex;
}
#result_table {
    width: 80%;
    border-collapse: collapse;
    border: 1px solid #777;
}
#header_row {
    font-weight: bold;
    color: white;
    background-color: #777;
}
#result_table td {
    border: 1px solid #777;
    padding: 2px;
}
#total_row  { font-weight: bold; }
.passClass  { background-color: #6c6; }
.failClass  { background-color: #c60; }
.errorClass { background-color: #c00; }
.passCase   { color: #6c6; }
.failCase   { color: #c60; font-weight: bold; }
.errorCase  { color: #c00; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }


/* -- ending ---------------------------------------------------------------------- */
#ending {
}

</style>

</head>
<body>
<script language="javascript" type="text/javascript"><!--
output_list = Array();

/* level - 0:Summary; 1:Failed; 2:All */
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level < 1) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level > 1) {
                tr.className = '';
            }
            else {
                tr.className = 'hiddenRow';
            }
        }
    }
}


function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        tid0 = 't' + cid.substr(1) + '.' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        if (toHide) {
            document.getElementById('div_'+tid).style.display = 'none'
            document.getElementById(tid).className = 'hiddenRow';
        }
        else {
            document.getElementById(tid).className = '';
        }
    }
}


function showTestDetail(div_id){
    var details_div = document.getElementById(div_id)
    var displayState = details_div.style.display
    // alert(displayState)
    if (displayState != 'block' ) {
        displayState = 'block'
        details_div.style.display = 'block'
    }
    else {
        details_div.style.display = 'none'
    }
}


function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}

/* obsoleted by detail in <div>
function showOutput(id, name) {
    var w = window.open("", //url
                    name,
                    "resizable,scrollbars,status,width=800,height=450");
    d = w.document;
    d.write("<pre>");
    d.write(html_escape(output_list[id]));
    d.write("\n");
    d.write("<a href='javascript:window.close()'>close</a>\n");
    d.write("</pre>\n");
    d.close();
}
*/
--></script>

<div class='heading'>
<h1>测试报告</h1>
<p class='attribute'><strong>Start Time:</strong> 2022-05-05 22:29:23</p>
<p class='attribute'><strong>Duration:</strong> 0:00:01.040001</p>
<p class='attribute'><strong>Status:</strong> Error 5</p>

<p class='description'>测试报告描述</p>
</div>



<p id='show_detail_line'>Show
<a href='javascript:showCase(0)'>Summary</a>
<a href='javascript:showCase(1)'>Failed</a>
<a href='javascript:showCase(2)'>All</a>
</p>
<table id='result_table'>
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row'>
    <td>Test Group/Test case</td>
    <td>Count</td>
    <td>Pass</td>
    <td>Fail</td>
    <td>Error</td>
    <td>View</td>
</tr>

<tr class='errorClass'>
    <td>test_register.TestRegister</td>
    <td>5</td>
    <td>0</td>
    <td>0</td>
    <td>5</td>
    <td><a href="javascript:showClassDetail('c1',5)">Detail</a></td>
</tr>

<tr id='ft1.1' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_success_00001_手机号为空</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.1')" >
        error</a>

    <div id='div_ft1.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.1: Traceback (most recent call last):
  File "D:\python\lib\site-packages\urllib3\connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "D:\python\lib\site-packages\urllib3\util\connection.py", line 95, in create_connection
    raise err
  File "D:\python\lib\site-packages\urllib3\util\connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\python\lib\site-packages\urllib3\connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "D:\python\lib\site-packages\urllib3\connectionpool.py", line 398, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "D:\python\lib\site-packages\urllib3\connection.py", line 239, in request
    super(HTTPConnection, self).request(method, url, body=body, headers=headers)
  File "D:\python\lib\http\client.py", line 1252, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "D:\python\lib\http\client.py", line 1298, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "D:\python\lib\http\client.py", line 1247, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "D:\python\lib\http\client.py", line 1026, in _send_output
    self.send(msg)
  File "D:\python\lib\http\client.py", line 966, in send
    self.connect()
  File "D:\python\lib\site-packages\urllib3\connection.py", line 205, in connect
    conn = self._new_conn()
  File "D:\python\lib\site-packages\urllib3\connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: &lt;urllib3.connection.HTTPConnection object at 0x0000000002E1F148&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\python\lib\site-packages\requests\adapters.py", line 450, in send
    timeout=timeout
  File "D:\python\lib\site-packages\urllib3\connectionpool.py", line 786, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "D:\python\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /futureloan/login (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000000002E1F148&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 42, in test_register_success
    headers=json.loads(test_data['headers']))
  File "D:\code\api_framework_v1\common\requests_handler.py", line 9, in visit
    res = self.session.request(method, url, params=params, data=data, json=json, headers=headers, **kwargs)
  File "D:\python\lib\site-packages\requests\sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\python\lib\site-packages\requests\sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "D:\python\lib\site-packages\requests\adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /futureloan/login (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000000002E1F148&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
Traceback (most recent call last):
  File "D:\python\lib\site-packages\urllib3\connection.py", line 175, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "D:\python\lib\site-packages\urllib3\util\connection.py", line 95, in create_connection
    raise err
  File "D:\python\lib\site-packages\urllib3\util\connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\python\lib\site-packages\urllib3\connectionpool.py", line 710, in urlopen
    chunked=chunked,
  File "D:\python\lib\site-packages\urllib3\connectionpool.py", line 398, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "D:\python\lib\site-packages\urllib3\connection.py", line 239, in request
    super(HTTPConnection, self).request(method, url, body=body, headers=headers)
  File "D:\python\lib\http\client.py", line 1252, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "D:\python\lib\http\client.py", line 1298, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "D:\python\lib\http\client.py", line 1247, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "D:\python\lib\http\client.py", line 1026, in _send_output
    self.send(msg)
  File "D:\python\lib\http\client.py", line 966, in send
    self.connect()
  File "D:\python\lib\site-packages\urllib3\connection.py", line 205, in connect
    conn = self._new_conn()
  File "D:\python\lib\site-packages\urllib3\connection.py", line 187, in _new_conn
    self, "Failed to establish a new connection: %s" % e
urllib3.exceptions.NewConnectionError: &lt;urllib3.connection.HTTPConnection object at 0x0000000002E1F148&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\python\lib\site-packages\requests\adapters.py", line 450, in send
    timeout=timeout
  File "D:\python\lib\site-packages\urllib3\connectionpool.py", line 786, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "D:\python\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /futureloan/login (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000000002E1F148&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 42, in test_register_success
    headers=json.loads(test_data['headers']))
  File "D:\code\api_framework_v1\common\requests_handler.py", line 9, in visit
    res = self.session.request(method, url, params=params, data=data, json=json, headers=headers, **kwargs)
  File "D:\python\lib\site-packages\requests\sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "D:\python\lib\site-packages\requests\sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "D:\python\lib\site-packages\requests\adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /futureloan/login (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x0000000002E1F148&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.2' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_success_00002_密码为空</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.2')" >
        error</a>

    <div id='div_ft1.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.2: Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 40, in test_register_success
    config.host + test_data['url'],
TypeError: can only concatenate str (not "NoneType") to str
Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 40, in test_register_success
    config.host + test_data['url'],
TypeError: can only concatenate str (not "NoneType") to str


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.3' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_success_00003_None</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.3')" >
        error</a>

    <div id='div_ft1.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.3: Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 40, in test_register_success
    config.host + test_data['url'],
TypeError: can only concatenate str (not "NoneType") to str
Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 40, in test_register_success
    config.host + test_data['url'],
TypeError: can only concatenate str (not "NoneType") to str


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.4' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_success_00004_None</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.4')" >
        error</a>

    <div id='div_ft1.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.4: Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 40, in test_register_success
    config.host + test_data['url'],
TypeError: can only concatenate str (not "NoneType") to str
Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 40, in test_register_success
    config.host + test_data['url'],
TypeError: can only concatenate str (not "NoneType") to str


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.5' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_success_00005_None</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.5')" >
        error</a>

    <div id='div_ft1.5' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.5').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.5: Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 40, in test_register_success
    config.host + test_data['url'],
TypeError: can only concatenate str (not "NoneType") to str
Traceback (most recent call last):
  File "D:\code\api_framework_v1\libs\ddt.py", line 191, in wrapper
    return func(self, *args, **kwargs)
  File "D:\code\api_framework_v1\test_cases\test_register.py", line 40, in test_register_success
    config.host + test_data['url'],
TypeError: can only concatenate str (not "NoneType") to str


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='total_row'>
    <td>Total</td>
    <td>5</td>
    <td>0</td>
    <td>0</td>
    <td>5</td>
    <td>&nbsp;</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>

</body>
</html>
